packages/cli-platform-android/src/commands/runAndroid/index.ts

```diff
@@ -61,7 +61,7 @@ async function runAndroid(_argv: Array<string>, config: Config, args: Flags) {
     );
 
     if (startPackager) {
-      await startServerInNewWindow(
+      startServerInNewWindow(
```

Comment:
mind also adding comment why awaiting this function breaks things?

Reply:
Done.